docs/using.rst

```diff
@@ -105,47 +113,57 @@ To use ADB with Host-Based tools
-
+* Docker is a default service for ADB boxes and does not require any configuration to ensure it is started.
+  Additionally, Red Hat Enterprise Linux Container Development Kit boxes, which
+  are based on the Atomic Developer Bundle, automatically start OpenShift as well.
+* You can enable multiple services as a comma separated list. For instance: `docker, openshift`.
 
 3. Enable any specific options for the services you have selected as:
 
- * OpenShift: Specific versions can be specified using the following variables:
+   For instance, in OpenShift, specific versions can be specified using the following variables:
 
    1. `config.servicemanager.openshift_docker_registry = "docker.io"` - Specifies the registry from where the service should be pulled.
    2. `config.servicemanager.openshift_image_name = "openshift/origin"` - Specifies the image to be used.
    3. `config.servicemanager.openshift_image_tag = "v1.1.1"` - Specifies the version of the image to be used.
 
-
 4. Start the ADB using ``vagrant up``. For details consult the `Installation documentation`_.
 
 .. _Installation documentation: https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst
 
-5. Configure the environment and download the required TLS certificates using the plugin.
-   The example below shows the command and the output for Linux and Mac OS X. On Microsoft Windows the output may vary depending on the execution environment.::
-
-    	$ vagrant service-manager env
-    	Configured services:
-    	docker - running
-        openshift - stopped
-    	kubernetes - stopped
-        docker env:
-    	# Set the following environment variables to enable access to the
-    	# docker daemon running inside of the vagrant virtual machine:
-    	export DOCKER_HOST=tcp://172.28.128.182:2376
-    	export DOCKER_CERT_PATH=/home/pchandra/test_adb/.vagrant/machines/default/libvirt/docker
-    	export DOCKER_TLS_VERIFY=1
-    	export DOCKER_API_VERSION=1.20
-    	# run following command to configure your shell:
-    	# eval "$(vagrant service-manager env docker)"
+
+5. Configure the environment and download the required TLS certificates using
+   the plugin. The example below shows the command and the output for Linux and Mac OS X.
+   On Microsoft Windows the output may vary depending on the execution environment::
+
+     $ vagrant service-manager env
+     # docker env:
+     # Set the following environment variables to enable access to the
+     # docker daemon running inside of the vagrant virtual machine:
+     export DOCKER_HOST=tcp://10.1.2.2:2376
+     export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+     export DOCKER_TLS_VERIFY=1
+     export DOCKER_API_VERSION=1.21
+
+     # run following command to configure your shell:
+     # eval "$(vagrant service-manager env)"
 
    Setting these environment variables allows programs, such as Eclipse and the
```

Comment:
I believe it is "allows" if the sentence starts with "Setting".  Delete the words setting and allow is good.